website/docs/installation/windows-install/index.md

```diff
@@ -1,28 +1,124 @@
 ---
 sidebar_position: 1
 title: Windows
-description: How to install Podman on Windows.
+description: How to install Podman Desktop and Podman on Windows.
 tags: [podman-desktop, installing, windows]
 keywords: [podman desktop, containers, podman, installing, installation, windows]
 ---
 
-# Installing Podman Desktop using the Windows installer
+# Installing Podman Desktop and Podman on Windows
 
-Consider using this installation method if you don't require another method such as:
+## Installing Podman Desktop
 
-- [Installing silently with the Windows installer](/docs/installation/windows-install/installing-podman-desktop-silently-with-the-windows-installer)
-- [Installing with Chocolatey](/docs/installation/windows-install/installing-podman-desktop-with-chocolatey)
-- [Installing with Scoop](/docs/installation/windows-install/installing-podman-desktop-with-scoop)
-- [Installing with Winget](/docs/installation/windows-install/installing-podman-desktop-with-winget)
-- [Installing Podman Desktop and Podman in a restricted environment](/docs/installation/windows-install/installing-podman-desktop-and-podman-in-a-restricted-environment)
+To install Podman Desktop:
 
-#### Procedure
+1. [Download the Windows installer](/downloads/windows).
+
+1. To start the Podman Desktop installer, open the downloaded file.
+
+   ![Podman Desktop Setup installing](img/podman-desktop-setup-installing.png)
+
+<details>
+<summary>
+Alternatively, when your environment requires another installation method, consider installing with:
```

Comment:
elongated sentence, maybe just "Alternative installation methods:" ?

Reply:
That would not explain when you should consider the alternate methods: because you have environmental constraints, not for the sake of choosing an alternate method.

Reply:
```suggestion
Alternate installation methods:
```